includes/api/class-edd-api.php

```diff
@@ -2267,4 +2269,25 @@ public function log_requests() {
 		return apply_filters( 'edd_api_log_requests', true );
 	}
 
+	/**
+	 * Check API keys vs token
+	 *
+	 * @access public
+	 * @since  2.8.2
+	 *
+	 * @param string $secret Secret key
+	 * @param string $public Public key
+	 * @param string $token Token used in API request
+	 *
+	 * @return bool
+	 */
+	public function check_keys( $secret, $public, $token ){
+		if ( hash_equals( md5( $secret . $public ), $token ) ) {
```

Comment:
Can we go ahead and make this a single line? `return hash_equals( md5( $secret . $public ), $token )`.